mlrun/utils/v3io_clients.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, FrozenSet
+
+from v3io.dataplane import Client as V3IOClient
+from v3io_frames import Client as FramesClient
+
+_v3io_clients: Dict[FrozenSet, V3IOClient] = {}
+_frames_clients: Dict[FrozenSet, FramesClient] = {}
+
+
+# TODO: Fix type hints for frames and v3io clients
```

Comment:
Any reason not doing it ?
(what I understand from it is that you simply need to add `-> FramesClient` and ... for the v3io...)

Reply:
Unfortunately `from v3io_frames import Client` > `Client` is actually a factory function that returns an abstract Client object. This should be first addressed in v3io_frames. And once it's changed there it should updated here.